test/system/src/test/java/io/pravega/test/system/AbstractScaleTests.java

```diff
@@ -37,10 +37,12 @@
     private final ConnectionFactory connectionFactory = new ConnectionFactoryImpl(false);
     @Getter(lazy = true)
     private final ClientFactory clientFactory = new ClientFactoryImpl(SCOPE, new ControllerImpl(getControllerURI(),
-            ControllerImplConfig.builder().retryAttempts(1).build(), getConnectionFactory().getInternalExecutor()));
+            ControllerImplConfig.builder().retryAttempts(12).maxBackoffMillis(5000).build(), getConnectionFactory()
```

Comment:
Let's just rely on the default, which is 10. I suggest to remove that `retryAttempts()` call altogether.

Reply:
Removed it.